README.md

```diff
@@ -103,8 +107,19 @@ for more detail on querying.
 
 ## Optional
 
-An alternative to using `curl` for ES config and querying is
-[kibana](https://www.elastic.co/products/kibana)
+An alternative to using `curl` for ES config, querying and also for visualisation is [kibana](https://www.elastic.co/products/kibana).
+For the latest version use `brew install kibana`. Similarly you can use `apt-get` or `yum` for other platforms.
```

Comment:
A link to the official Kibana installation page would be better https://www.elastic.co/guide/en/kibana/current/install.html